spacetemplate/assets/agile.yaml

```diff
@@ -166,9 +170,11 @@ work_item_types:
   extends: *agileCommonTypeID
   name: Story
   can_construct: yes
-  description: 
-    A user story describes an executable chunk of a feature 
-    based on user experience.
+  description:
+    Incremental work or requirements identified to deliver a product feature.
+    Typically, it is expressed from the user’s perspective in the following
+    format - As a *user*, I want to achieve an *objective*, so that this
```

Comment:
What do you expect is rendered when something is enclosed in stars?

Reply:
I've removed the `*`s in there.

Reply:
I was trying to render them in Italics as those words can be replaced with the type of user, the objective and the purpose depending on the user. Something like <user_name>